set.mm

```diff
@@ -403908,13 +404004,22 @@ in common (for j=1, ... , k).  In contrast to the definition in Aksoy et
   $( Lemma for ~ wlkres .  TODO-AV:  Will become obsolete if
      ` H = ( F |`` ( 0 ..^ N ) ) ` is replaced by
      ` H = ( F prefix N ) ` in ~ wlkres and ~ trlres .  (Contributed by AV,
-     5-Mar-2021.) $)
+     5-Mar-2021.) (Proof shortened by AV, 3-Nov-2022.) $)
   wlkreslem0 $p |- ( ( F e. Word S /\ N e. ( 0 ... ( # ` F ) ) )
        -> ( # ` ( F |` ( 0 ..^ N ) ) ) = N ) $=
+    ( cword wcel cc0 chash cfv co wa cfzo cres cpfx pfxres eqcomd fveq2d pfxlen
+    cfz eqtrd ) BADECFBGHRIEJZBFCKILZGHBCMIZGHCTUAUBGTUBUAABCNOPABCQS $.
+
+  $( Obsolete proof of ~ wlkreslem0 as of 12-Oct-2022.  (Contributed by AV,
+     5-Mar-2021.)  (New usage is discouraged.)
+     (Proof modification is discouraged.) $)
+  wlkreslem0OLD $p |- ( ( F e. Word S /\ N e. ( 0 ... ( # ` F ) ) )
+       -> ( # ` ( F |` ( 0 ..^ N ) ) ) = N ) $=
     ( cword wcel cc0 chash cfv cfz co wa cfzo csubstr swrd0valOLD eqcomd fveq2d
     cres cop swrd0lenOLD eqtrd ) BADECFBGHIJEKZBFCLJQZGHBFCRMJZGHCUAUBUCGUAUCUB
     ABCNOPABCST $.
 
+
```

Comment:
no double blank lines except to separate sections

Reply:
Done (one blank line removed)